modules/varnish/templates/default.vcl.erb

```diff
@@ -13,6 +13,7 @@
 
 acl purge_acl {
   "localhost";
+  "10.1.0.0"/16;
```

Comment:
I'm not too familiar with the syntax, but the `/16` is fine outside the quotes?

Reply:
I wouldn't normally write it like that, but I was basing it off the varnish docs:
https://varnish-cache.org/docs/4.1/users-guide/purging.html#http-purging

Reply:
Weird!